crypter/vault.go

```diff
@@ -31,10 +33,34 @@ type VaultCreds struct {
 	RootToken string `json:"root_token"`
 }
 
-var status struct {
+var Status struct {
```

Comment:
Why is this exported?

Reply:
good question, I exported the status function and apparently got a bit export happy